src/components/carousel/carousel.js

```diff
@@ -506,6 +506,7 @@ class Carousel extends React.Component {
 
           <CSSTransitionGroup
             component='div'
+            style={ { height: '100%' } }
```

Comment:
Why don't you use a CSS class instead of an inline style property?

Reply:
The only reason was that it's a small correction to the styling, I didn't think it needed a css class, unless you think it's necessary.

Reply:
If I'm not wrong, the general case is to use a CSS class instead of an inline style (Inline styles should be used only to add dynamic content ¿?).
I really don't care if you use one or the other, I just wonder in which cases we can use the inline style.

Reply:
IMO it should go into `/src/components/carousel/carousel.scss`